shielddash/urls.py

```diff
@@ -17,10 +17,11 @@
 from django.contrib import admin
 from django.views.generic import TemplateView
 
-from .studies.views import StudyListView
+from .studies.views import StudyListView, study_detail
 
 
 urlpatterns = [
+    url(r'^studies/(?P<study_id>\d+)/', study_detail, name='study-detail'),
     url(r'^studies/', StudyListView.as_view(), name='study-list'),
```

Comment:
No, you only get multiple views out of a class if you're using the viewset/router stuff.